sources/packages/backend/apps/queue-consumers/src/processors/schedulers/sfas-integration/_tests_/sfas-integration.scheduler.e2e-spec.ts

```diff
@@ -25,6 +26,8 @@ const SFAS_LEGACY_RESTRICTION_FILENAME =
   "SFAS-TO-SIMS-2024MAR07-LEGACY-RESTRICTIONS.txt";
 const SFAS_ALL_RESTRICTIONS_FILENAME =
   "SFAS-TO-SIMS-2024MAR07-ALL-RESTRICTIONS.txt";
+const SFAS_SAIL_DATA_FILENAME =
+  "SFAS-TO-SIMS-2024MAR21-PY-MAXIMUM-DATA-IMPORT.txt";
```

Comment:
I would say not to use PY maximum in the file name here. It is a PT applications import.

Reply:
I'll go with 'SFAS-TO-SIMS-2024MAR21-PT-APPLICATION-DATA-IMPORT.txt" then.